js/services/engine.ts

```diff
@@ -882,10 +940,15 @@ export const initializeSimple = (
     }
 
     if (window.navigator.language) {
```

Comment:
Would be good to extract this duplication into a function.

Reply:
i didnt create the duplicate functions just added in the clause to use the new culture constant to the 2 existing ones. someone may want to go through and refactor that bit of the code at some time.

Reply:
I think that as the duplication has grown it is more of a concern. The problem is that if an issue is identified in a section of this code and is fixed, or is improved in some way, the duplicated section could easily get missed. This probably should have been addressed when first written but as we have an opportunity to improve it now I think we should.